charts/prometheus/templates/alertmanager/pvc.yaml

```diff
@@ -25,6 +25,7 @@ spec:
 {{- if .Values.alertmanager.persistentVolume.volumeBindingMode }}
   volumeBindingModeName: "{{ .Values.alertmanager.persistentVolume.volumeBindingMode }}"
 {{- end }}
+  allowVolumeExpansion: {{ .Values.alertmanager.persistentVolume.allowVolumeExpansion | default "false" | quote }}
```

Comment:
@bseenu  does this work on a pvc I though only on a storage class.
You can only expand a PVC if its storage class's allowVolumeExpansion field is set to true.
```
apiVersion: storage.k8s.io/v1
kind: StorageClass
metadata:
  name: gluster-vol-default
provisioner: kubernetes.io/glusterfs
parameters:
  resturl: "http://192.168.10.100:8080"
  restuser: ""
  secretNamespace: ""
  secretName: ""
allowVolumeExpansion: true